<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeviewDock</class>
 <widget class="QDockWidget" name="TreeviewDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="format-indent-more">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="windowTitle">
   <string>Tree</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QTreeView" name="treeView">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="editButton">
      <property name="text">
       <string>Enable edit</string>
      </property>
      <property name="icon">
       <iconset theme="tools-check-spelling">
        <normaloff>.</normaloff>.</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>297</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="fopenButton">
      <property name="text">
       <string>Open item in external program</string>
      </property>
      <property name="icon">
       <iconset theme="edit-find">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QToolButton" name="dopenButton">
      <property name="text">
       <string>Open item direcotry in explorer</string>
      </property>
      <property name="icon">
       <iconset theme="document-open">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
